crates/solidity/outputs/cargo/crate/src/backend/built_ins.rs

```diff
@@ -296,6 +299,7 @@ impl<'a> BuiltInsResolver<'a> {
             }
             "lt" => Some(BuiltIn::YulLt),
             "mcopy" if self.language_version >= VERSION_0_8_24 => Some(BuiltIn::YulMcopy),
+            "clz" if self.language_version >= VERSION_0_8_31 => Some(BuiltIn::YulClz),
```

Comment:
nit: should this be added earlier to keep the list sorted?

Reply:
Sorted!

_pun intended_